app/models/user/permissions.rb

```diff
@@ -16,15 +16,12 @@ module User::Permissions
     alias_attribute :allowed_service_ids, :member_permission_service_ids
   end
 
-  #TODO: this is repeated from bcms_hacks plugins because of some loading problem
```

Comment:
This bcms ("browser CMS") thing was removed long time ago, so this comment is not relevant anymore.